_includes/api/en/4x/req-fresh.md

```diff
@@ -2,12 +2,19 @@
 
 When the response is still "fresh" in the client's cache `true` is returned, otherwise `false` is returned to indicate that the client cache is now stale and the full response should be sent.
 
+In order to check whether the response is still "fresh" in the client's cache or not, two tags will be used ETag and If-None-Match. ETag will be fetched from response (saved internally from application) and other tag will be received from client's request.
```

Comment:
Hey @gulgulia17, 
@crandmck said I could leave any feedback for your PR. I'm not an maintainer here or anything, so you don't have to listen to me :]. It's all just punctuation related stuff. 
1. Add a colon to this:
`In order to check whether the response is still "fresh" in the client's cache or not, two tags will be used: ETag and If-None-Match.`
2. Add a comma in the middle + missing 'the':
`ETag will be fetched from response (saved internally from application), and the other tag will be received from client's request.`